helix-core/src/test/java/org/apache/helix/ZkTestHelper.java

```diff
@@ -146,6 +146,70 @@ public void process(WatchedEvent event) {
     LOG.info("After expiry. sessionId: " + Long.toHexString(curZookeeper.getSessionId()));
   }
 
+  public static void expireSharedZkClientSession(HelixZkClient client) throws Exception {
```

Comment:
Naming *Shared*ZkClient doesn’t match the parameter type HelixZkClient.

FYI, it seems the only difference between expireSession and this is, this does not have handleNewSession implementation logic. Would you consider reducing the duplicate code?

Reply:
It is different for a reason. Shared ZkClient does not have handleNewSession. Will combine see the above discussion with JJ.